src/osd/PrimaryLogPG.cc

```diff
@@ -2070,7 +2070,7 @@ void PrimaryLogPG::do_op(OpRequestRef& op)
   }
 
   ObjectContextRef obc;
-  bool can_create = op->may_write() || op->may_cache();
+  bool can_create = op->may_write();
```

Comment:
we might want to backport this if this is a bug fix.

Reply:
I think it's pretty harmless... I'd rather not backport it for risk of some unforeseen side-effect.